examples/platform/efr32/rs911x/hal/rsi_hal.h

```diff
@@ -54,6 +54,12 @@
 // Macro to drive high value on GPIO
 #define RSI_HAL_GPIO_HIGH 1
 
+// Macro to drive LDMA channel number
+#define RSI_LDMA_TRANSFER_CHANNEL_NUM 3
+
+// Macro to drive semaphore block minimun tick value
```

Comment:
Since you're using `pdMS_TO_TICKS` both the comment and define should be rename to MS instead of ticks

Reply:
Thanks for the suggestion, made the changes according your comment.